data/tlg1463/tlg001/tlg1463.tlg001.opp-grc1.xml

```diff
@@ -72,17 +72,18 @@
    </editor>
 	  <editor>
 	<persName>
-	<name xml:lang="deu">Ludwig Radermacher</name>
```

Comment:
Why did you delete these two lang-tags here and in line 81? Aren't they correct?

Reply:
I guess, we don't need them.